install/kubernetes/cilium/templates/_helpers.tpl

```diff
@@ -141,3 +141,17 @@ Enable automatic lookup of k8sServicePort from the cluster-info ConfigMap (kubea
   {{- end }}
 {{- end }}
 
+{{/*
+Return user specify envoy.enabled or default value based on the upgradeCompatibility
+*/}}
+{{- define "envoyDeamonSetEnabled" }}
```

Comment:
```suggestion
{{- define "envoyDaemonSetEnabled" }}
```
Sorry, blame Unix people in the 70s for using an archaic spelling...

Reply:
oops, good catch :face_holding_back_tears: